### PR TITLE
Add deb plugin repo notes to 1.5

### PR DESCRIPTION
**Pull request by GregSutcliffe**
Also add some generic repo notes to the older versions

### CODE REVIEW

File: _includes/manuals/1.5/2.1_quickstart_installation.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ For **Debian variants**, run this (replace 'wheezy' with 'precise' if on Ubuntu
 
 {% highlight bash %}
 echo "deb http://deb.theforeman.org/ wheezy {{page.version}}" > /etc/apt/sources.list.d/foreman.list
+echo "deb http://deb.theforeman.org/ plugins {{page.version}}" > /etc/apt/sources.list.d/foreman.list
```
Comment (domcleal) [inline _includes/manuals/1.5/2.1_quickstart_installation.md]: `>>` surely?  ditto below

File: _includes/manuals/1.5/3.3.3_debian_packages.md [hunk 2]
```diff
@@ -39,6 +39,14 @@ deb http://deb.theforeman.org/ squeeze nightly
 deb http://deb.theforeman.org/ precise nightly
 {% endhighlight %}
 
+You may also want some plugins from the plugin repo (required for the Foreman Installer):
+{% highlight bash %}
+# Stable
+deb http://deb.theforeman.org/ plugins {{page.version}}
+# Latest versions
```
Comment (domcleal) [inline _includes/manuals/1.5/3.3.3_debian_packages.md]: I'd change this to "Plugins compatible with Foreman nightly" or similar so the compatibility issue is clear.

File: _includes/manuals/1.5/2.1_quickstart_installation.md (hunk 1 above)
Comment (GregSutcliffe) [inline _includes/manuals/1.5/2.1_quickstart_installation.md]: Ooops, fixed

File: _includes/manuals/1.5/3.3.3_debian_packages.md (hunk 2 above)
Comment (GregSutcliffe) [inline _includes/manuals/1.5/3.3.3_debian_packages.md]: Good thinking, upated

**Comment by domcleal** [comment]
Merged, thanks.  .bundle sent to gh-pages instead.

**Pull request closed without merging**